pkg/doc/signature/proof/data.go

```diff
@@ -86,14 +110,15 @@ const (
 	proofType excludedKey = iota + 1
 	proofID
 	proofValue
+	jws
 )
 
 func (ek excludedKey) String() string {
-	return [...]string{"type", "id", "proofValue"}[ek-1]
+	return [...]string{"type", "id", "proofValue", "jws"}[ek-1]
 }
 
 func excludedKeyFromString(s string) excludedKey {
-	for _, ek := range [...]excludedKey{proofType, proofID, proofValue} {
+	for _, ek := range [...]excludedKey{proofType, proofID, proofValue, jws} {
```

Comment:
And this one as well under the const definition `[...]excludedKey{proofType, proofID, proofValue, jws} `